ansible/roles/polkadot-validator/templates/polkadot.service.j2

```diff
@@ -1,47 +1,71 @@
+{# --- PREPARE VARIABLES #}
+{% set nodeName = hostvars[inventory_hostname].nodeName|default(None) %}
+{% set execution = hostvars[inventory_hostname].execution|default(None) %}
+{% set wasmExecution = hostvars[inventory_hostname].wasmExecution|default(None) %}
+{% set telemetryUrl = hostvars[inventory_hostname].telemetryUrl|default(None) %}
+{% set enableReverseProxy = hostvars[inventory_hostname].enableReverseProxy|default(None) %}
+{% set loggingFilter = hostvars[inventory_hostname].loggingFilter|default(None) %}
+{% set basePath = hostvars[inventory_hostname].basePath|default(None) %}
+{% set additionalFlags = hostvars[inventory_hostname].additionalFlags|default(None) %}
+{# ## Legacy/deprecated flags #}
+{% if wasmExecution is none %}
+  {% set wasmExecution = hostvars[inventory_hostname].wasm_execution|default(None) %}
+{% endif %}
+{% if basePath is none %}
```

Comment:
an handle is probably necessary even for this situation: https://github.com/w3f/polkadot-validator-setup/blob/master/ansible/roles/polkadot-restore-db/tasks/main.yml#L7-L15